.github/pull_request_template.md

```diff
@@ -2,4 +2,4 @@ Please ensure this checklist is followed for any pull requests for this repo. Th
 * [ ] Relevant documentation for this PR has to be completed and reviewed by @mdr0id before the PR can be merged
 * [ ] A test plan for the PR must be documented in the PR notes and included in the test plan for the next regular release
```

Comment:
We need some documentation about how these test plans are intended to be written - I don't know of even any examples - and it's not clear how to include it in the test plan.